aci/resource_aci_rtctrlattrp.go

```diff
@@ -124,7 +124,7 @@ func resourceAciActionRuleProfileCreate(ctx context.Context, d *schema.ResourceD
 	if NameAlias, ok := d.GetOk("name_alias"); ok {
 		rtctrlAttrPAttr.NameAlias = NameAlias.(string)
 	}
-	rtctrlAttrP := models.NewActionRuleProfile(fmt.Sprintf("attr-%s", name), TenantDn, desc, rtctrlAttrPAttr)
+	rtctrlAttrP := models.NewActionRuleProfile(fmt.Sprintf("attr-%s", name), TenantDn, desc, "", rtctrlAttrPAttr)
 
```

Comment:
This should be fixed on latest master. Remove that change and rebase on top of master.